keyboards/handwired/dactyl_manuform/5x7_2_6/readme.md

```diff
@@ -0,0 +1,29 @@
+# handwired/dactyl_manuform/5x7_2_6
+
+![Planck](https://raw.githubusercontent.com/ngrigoriev/dactyl-manuform-5x7_2_6/main/pics/kbpic_small_21.jpg)
```

Comment:
Please rehost this image on Imgur. Also, "Planck" doesn't seem right...
```suggestion
![keyboad_image](https://raw.githubusercontent.com/ngrigoriev/dactyl-manuform-5x7_2_6/main/pics/kbpic_small_21.jpg)
```

Reply:
Hi, sorry, I refuse to register on Imgur because it requires a phone number. Any other service that respects the privacy of the users will be fine. If usage of Imgur is a requirement here, please reject this PR.

Reply:
No worries.